WebApp/src/main/java/com/atroshonok/controller/ProductController.java

```diff
@@ -379,6 +382,7 @@ private void setRequestCurrentPage(HttpServletRequest request, ClientFilter clie
     }
 
     // Returns the list of the page numbers for the pagination
+	// do you really need the list for number of pages only only 1 number - the count of it?

```

Comment:
// i need it for UI. I use it in <c:forEach> tag at my jsp page for displaing a list of digits